### PR TITLE
[Fixes #5168]Add info for WinRm config with DSC

### LINKED ISSUE #5168: Does not work
I copied the statements as they are in the page and at the command Start-DscConfiguration I get an error "The client cannot connect to the destination specified in the request". 
Troubleshooting as first steps is not the best way to encourage using DSC!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6899bee9-ef7a-7447-0807-62ab30ae9606
* Version Independent ID: 6160bf0d-b5bb-003d-df4c-a42e590d3866
* Content: [Get started with Desired State Configuration (DSC) for Windows - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/dsc/getting-started/wingettingstarted?view=powershell-6#feedback)
* Content Source: [reference/docs-conceptual/dsc/getting-started/winGettingStarted.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/dsc/getting-started/winGettingStarted.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary
Adds not about updating WinRm configuration prior to using DSC

## PR Context
Fixes #5168 
Fixes [AB#1655020](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1655020)

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [x] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/dsc/quickstarts/website-quickstart.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ The `Start-DscConfiguration` cmdlet tells the [Local Configuration Manager (LCM)
 which is the engine of DSC, to apply the configuration.
 The LCM does the work of calling the DSC resources to apply the configuration.
 
+> [!NOTE] To allow DSC to run, Windows needs to be configured to receive PowerShell remote commands,
```
Comment (sdwheeler) [inline reference/docs-conceptual/dsc/quickstarts/website-quickstart.md]: Reflow wrapped this incorrectly. You have to watch out for NOTE blocks.

File: reference/docs-conceptual/dsc/getting-started/winGettingStarted.md [hunk 2]
```diff
@@ -68,45 +68,63 @@ Configuration EnvironmentVariable_Path
...
-`Install-Module 'PSDscResources' -Verbose`
+```PowerShell
+Install-Module 'PSDscResources' -Verbose
+```
 
 #### Apply the configuration to the machine
 
-Configuration documents (MOF files) can be applied to the machine
-using the [Start-DscConfiguration](/powershell/module/psdesiredstateconfiguration/start-dscconfiguration) cmdlet.
+> [!NOTE] To allow DSC to run, Windows needs to be configured to receive PowerShell remote commands,
```
Comment (sdwheeler) [inline reference/docs-conceptual/dsc/getting-started/winGettingStarted.md]: Reflow wrapped this incorrectly.

Comment (chasewilson) [reply reference/docs-conceptual/dsc/getting-started/winGettingStarted.md]: Followed the errors there and wondered how in the world I did this and then remembered. Sigh, live and learn

File: reference/docs-conceptual/dsc/quickstarts/website-quickstart.md (hunk 1 above)
Comment (chasewilson) [reply reference/docs-conceptual/dsc/quickstarts/website-quickstart.md]: I wondered why I had written it that way and then realized what had happened. Lesson learned

File: reference/docs-conceptual/dsc/quickstarts/website-quickstart.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ title:  Quickstart - Create a website with DSC
 
 > Applies To: Windows PowerShell 4.0, Windows PowerShell 5.0
```
Comment (sdwheeler) [inline reference/docs-conceptual/dsc/quickstarts/website-quickstart.md]: Let's move this "Applies to" line after the H1 and before the first paragraph.

Comment (chasewilson) [reply reference/docs-conceptual/dsc/quickstarts/website-quickstart.md]: Done

**Pull request merged**